src/components/MAPI/permissions/utils.ts

```diff
@@ -271,6 +278,78 @@ export async function fetchPermissions(
   return permissions;
 }
 
+export async function fetchPermissionsAtClusterScope(
+  httpClientFactory: HttpClientFactory,
+  auth: IOAuth2Provider,
+  useCases: IPermissionsUseCase[],
+  namespacedPermissions: IPermissionMap,
+  user?: string,
+  groups?: string[]
+): Promise<IPermissionMap> {
+  const requests: authorizationv1.ISelfSubjectAccessReviewSpec[] = [
+    {
+      resourceAttributes: {
+        verb: 'list',
+        group: 'rbac.authorization.k8s.io',
+        resource: 'clusterrolebindings',
+      },
+    },
+    {
+      resourceAttributes: {
+        verb: 'get',
+        group: 'rbac.authorization.k8s.io',
+        resource: 'clusterroles',
+      },
+    },
+  ];
+
+  // Can the user LIST clusterrolebindings and GET clusterroles?
+  const accessReviewResponses = await Promise.allSettled(
+    requests.map((req) =>
+      authorizationv1.createSelfSubjectAccessReview(
+        httpClientFactory(),
+        auth,
+        req
+      )
+    )
+  );
+
+  for (const response of accessReviewResponses) {
+    if (
+      response.status === 'rejected' &&
+      !metav1.isStatusError(
+        (response.reason as GenericResponse).data,
+        metav1.K8sStatusErrorReasons.Forbidden
+      )
+    ) {
```

Comment:
I added this earlier, but on second thought the check doesn't make sense - whether or not the user has access to these resources is already handled by the next block. Here we should just be handling the rejected promises. Will make the changes